design/media_types.go

```diff
@@ -199,40 +199,40 @@ var User = a.MediaType("application/vnd.user+json", func() {
 	})
 })
 
-var searchPagingLinks = Type("searchPagingLinks", func() {
-	Attribute("prev", String)
-	Attribute("next", String)
-	Attribute("first", String)
-	Attribute("last", String)
+var searchPagingLinks = a.Type("searchPagingLinks", func() {
+	a.Attribute("prev", d.String)
```

Comment:
@tsmaeder yes I would agree with you on this. `dsl` and `api` much better than `a` and `d`. Still I am not very ok with the name `api`  as it is too generic (I don't have any suggestions either). `dsl` sounds perfect to use.